spk/sshfs/Makefile

```diff
@@ -0,0 +1,24 @@
+SPK_NAME = sshfs
+SPK_VERS = 2.5
+SPK_REV = 1
+SPK_ICON = src/sshfs.png
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = SSHFS is a filesystem client based on the SSH File Transfer Protocol
```

Comment:
See my other comment on lower case usage. Whatever the name, it should be the same throughout the package (as in DISPLAY_NAME, DNAME; everything a user of the package can come across during normal usage)